biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -51,64 +53,97 @@ export const SimulatePage = () => {
       return {
         fromItem: edge.v,
         toItem: edge.w,
+        // TODO this is not a correct value
+        powerThroughLinesKiloWatts: edge.value.power,
       };
     });
   };
 
+  const playSimulation = () => {
```

Comment:
Per UX principles, every action should be immediately un-doable by users. The "reset" button is helpful, but once they click "play", we have to give users a way to "stop" or "pause" the simulation as well.

Reply:
Added to issue #86